{"uuid":"78314c8d-09c1-4b8d-94b9-a74f24f48bda","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["125ed16f-4140-436f-8bd4-01b346e5fa2b"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673365586930,"stop":1673365586931}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673365586940,"stop":1673365586940},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673365587307,"stop":1673365587308},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673365587540,"stop":1673365587540},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"df2e79f2-303b-4b82-95fd-635862fb6adb-attachment","type":"image/png"},{"name":"Myscreenshot","source":"2e92e419-2b5d-455d-a536-f19f9e0942e9-attachment"}],"parameters":[],"start":1673365587556,"stop":1673365587925}],"start":1673365586929,"stop":1673365587926}